tests/python/test_externalforce.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+import tempfile
+import numpy as np
+from . import utils
+
+
+def test_external_force():
+    # Get the simulator
+    gazebo = utils.Gazebo(physics_rate=1000, iterations=1, rtf=100)
+
+    # Serialize the cube urdf
+    handle, model_file = tempfile.mkstemp()
+    with open(handle, 'w') as f:
+        f.write(utils.get_cube_urdf())
+
+    # Create the first cube and insert it in the simulation
+    cube = utils.CubeGazeboRobot(model_file=model_file,
+                                 gazebo=gazebo.simulator,
+                                 initial_position=np.array([0, 0, 1.0]))
+
```

Comment:
Just out of curiosity: Why has it been chosen to use serialization and then pass a file to `CubeGazeboRobot` in this case, and not pass directly the `utils.get_cube_urdf()` output?

Reply:
If you check https://github.com/robotology/gym-ignition/pull/124/files/af4c9d750f953f87da4eeede155e9ab9aa1ea43d#diff-2fa3603fe004643ef39d4d16c04a3003R132, the `get_cube_urdf` returnes a string. Robot objects like `CubeGazeboRobot` want a `model_file` argument containing a file name.

I didn't want to create an extra file stored in the repository only for the tests. Since the urdf in this case is trivial and the computation of different quantities like inertia is done on the fly, I preferred to store it as a string and serialize it to a temporary file when needed.

Reply:
Ok, I see. Thanks.